components/view_user_group_modal/view_user_group_modal.scss

```diff
@@ -0,0 +1,156 @@
+.modal {
```

Comment:
Again, the root class is not based on component name.

Reply:
Root class is now related to the component name